src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -469,7 +580,35 @@ public string EncryptToken(string innerJwt, EncryptingCredentials encryptingCred
             if (encryptingCredentials == null)
                 throw LogHelper.LogArgumentNullException(nameof(encryptingCredentials));
 
-            return EncryptTokenPrivate(innerJwt, encryptingCredentials, null);
+            return EncryptTokenPrivate(innerJwt, encryptingCredentials, null, null);
+        }
+
+        /// <summary>
+        /// Encrypts a JWS.
+        /// </summary>
+        /// <param name="innerJwt">A 'JSON Web Token' (JWT) in JWS Compact Serialization Format.</param>
```

Comment:
should we modify this parameter name and comment, we don't really know or verify that the string is a JWS in Compact Serialization Format.

Reply:
I agree that the parameter name and comment should be changed, or perhaps maybe we should verify that the string is in JWS format?

In the [RFC](https://tools.ietf.org/html/rfc7516#appendix-A.1.5) the value to be encrypted is referred to as 'plaintext', so we could call it that. 

However, if something other than a JWS is passed into this method, the result produced by this method will no longer be something our library can verify. I feel like this may cause confusion.

The name of the method also happens to be 'EncryptToken', which may also get confusing.